:globe_with_meridians: Update Turkish translation for `docs/tr/docs/fastapi-people.md` (#10547)
fastapi/fastapi: docs/tr/docs/fastapi-people.md

<div class="user-list user-list-center">
{% for user in people.maintainers %}

<div class="user"><a href="{{ user.url }}" target="_blank"><div class="avatar-wrapper"><img src="{{ user.avatarUrl }}"/></div><div class="title">@{{ user.login }}</div></a> <div class="count">Cevaplar: {{ user.answers }}</div><div class="count">Pull Request'ler: {{ user.prs }}</div></div>
{% endfor %}

</div>
{% endif %}

> Ben **FastAPI**'ı oluşturan geliştiricisiyim. Bununla ilgili daha fazla bilgiyi şurada okuyabilirsiniz:

Author: Benimde kulağıma pek doğru gelmiyor fakat tamamen doğru bir Türkçe kullanımı :D Belki şöyle bir şey daha uygun olabilir:
```suggestion
Ben **FastAPI**'ı oluşturan geliştiriciyim. Bununla ilgili daha fazla bilgiyi şurada okuyabilirsiniz:
```
Ben halen FastAPI geliştiricisi olduğunu vurgulamak istemiştim, böyle dediğimizde de yeterince belli oluyorsa öneriden ilerleyelim.

> Ben **FastAPI**'ı oluşturan geliştiricisiyim. Bununla ilgili daha fazla bilgiyi şurada okuyabilirsiniz:

Reviewer: ```suggestion
Ben **FastAPI**'ı oluşturan ve geliştiren kişiyim. Bununla ilgili daha fazla bilgiyi şurada okuyabilirsiniz:
```
Bu nasıl?

> Ben **FastAPI**'ı oluşturan geliştiricisiyim. Bununla ilgili daha fazla bilgiyi şurada okuyabilirsiniz:

Reviewer: Birde senin daha önce, "creator" için yaptığın iki çeviriye bakmak istiyorum:

https://github.com/tiangolo/fastapi/blob/3f3ee240dd8656962e94e89eceb3838508982068/docs/tr/docs/alternatives.md?plain=1#L34

Burada "yaratıcısı" demişiz.

https://github.com/tiangolo/fastapi/blob/3f3ee240dd8656962e94e89eceb3838508982068/docs/tr/docs/alternatives.md?plain=1#L286

Burada "geliştiricisi" demişiz.

---

[Tureng - maintainer - Türkçe İngilizce Sözlük](https://tureng.com/tr/turkce-ingilizce/maintainer) sayfasında gördüğüm kadarıyla `maintainer` aynı zamanda "geliştirici" demek için de kullanılıyor yani inanıyorum ki şöyle bir çeviri yeterli olacaktır...

```suggestion
Ben **FastAPI**'ın geliştiricisiyim. Bununla ilgili daha fazla bilgiyi şurada okuyabilirsiniz:
```

Türkçede "geliştirici" dendiğinde akla gelen şeylerden birisi de aslında "oluşturan" oluyor. Burada ki "maintainer" aynı zamanda "bakıcı" yani "sürekli ilgilenen" kişi anlamını taşıyor bunun için de "geliştirici" kullanılabilir mi? Evet çünkü geliştirici olmak illa "geçmişte oldu ve bitti" anlamını taşımaz, "geliştiricisiyim, hala geliştiriyorum"'da denilebilir...

Çok kafa yordum çözelim bunu 😆

> Ben **FastAPI**'ın geliştiricisiyim. Bununla ilgili daha fazla bilgiyi şurada okuyabilirsiniz:
>  [FastAPI yardım - yardım al -  benimle iletişime geç](help-fastapi.md#connect-with-the-author){.internal-link target=_blank}.

Reviewer: ```suggestion
Ben **FastAPI**'ın geliştiricisiyim. Bununla ilgili daha fazla bilgiyi şurada okuyabilirsiniz: [FastAPI yardım - yardım al -  benimle iletişime geç](help-fastapi.md#connect-with-the-author){.internal-link target=_blank}.
```

</div>
{% endif %}

> Ben **FastAPI**'nin yazarı ve geliştiricisiyim. Bununla ilgili daha fazla bilgiyi şurada okuyabilirsiniz:

Reviewer: ```suggestion
Ben **FastAPI**'nın yazarı ve geliştiricisiyim. Bununla ilgili daha fazla bilgiyi şurada okuyabilirsiniz:
```

 [FastAPI yardım - yardım al - Yazar ile iletişime geç](help-fastapi.md#connect-with-the-author){.internal-link target=_blank}.

> ... Burada size harika FastAPI topluluğunu göstermek istiyorum.

Reviewer: ```suggestion
...burada size harika FastAPI topluluğunu anlatmak istiyorum.
```

---

> **FastAPI**, topluluğundan çok destek alıyor. Ben de onların katkısını vurgulamak istiyorum.

Reviewer: ```suggestion
**FastAPI**, topluluğundan çok destek alıyor. Ben de onların katkılarını vurgulamak istiyorum.
```

Reviewer: https://github.com/tiangolo/fastapi/blob/6f6e786979eb3156b85e7c2f44d0b8af5cf0af17/docs/en/docs/fastapi-people.md?plain=1#L32

Reviewer: "onların katkısını" dediğimizde çıkan anlam ile "onların katkılarını" dediğimizde çıkan anlam farklı ancak burada söylenmek isteyen şey "farklı kişilerin, farklı katkıları: onların katkıları"

Author: Günün sonunda aralarında bir fark olduğunu düşünmüyorum, sonuçta hepsi günün sonunda FastAPI'a katkıda bulunuyor. Ancak 'katkılarını' demek gramer açısından daha doğru.

Reviewer: Birde şunu düşünüyorum, 182. satırda da "katkılarını" demişsin, orada ki anlam hemen hemen aynı. Sadece "onların" yok. Ayrıca burada bahsi geçen şey "bir kişinin birden fazla katkısı" yada "birden fazla kişinin tek katkısı" değil 😇.

> O mükemmel insanlar:

Reviewer: Orjinalinde "Mükemmel" vurgusu göremedim.

> * [GitHubdaki sorunları (issues) çözmelerinde diğerlerine yardım ediyor](help-fastapi.md#help-others-with-issues-in-github){.internal-link target=_blank}.

Reviewer: Outdated.

* [Pull Requests oluşturuyor](help-fastapi.md#create-a-pull-request){.internal-link target=_blank}.
* Pull Requests'leri gözden geçiriyorlar, [özellikle çeviriler için bu çok önemli](contributing.md#translations){.internal-link target=_blank}.

Onları bir alkışlayalım. 👏 🙇

## Geçen Ayın En Aktif Kullanıcıları

> Bunlar geçen ay boyunca [GitHub'da başkalarına en çok yardımcı olan](help-fastapi.md#help-others-with-issues-in-github){.internal-link target=_blank} kullanıcılar. ☕

Reviewer: ```suggestion
Geçtiğimiz ay boyunca [GitHub'da başkalarına en çok yardımcı olan](help-fastapi.md#help-others-with-issues-in-github){.internal-link target=_blank} kullanıcılar. ☕
```

Reviewer: `## Uzmanlar` başlığına kadar olan kısım outdated.
